File: Listas de Exercícios (Python)/Python/Lista 2 - Sala/5.py
print("Ordem crescente\n")

print("--Valores decimais devem ser digitados com (.)!--")

A=float(input("Valor 1: "))
B=float(input("Valor 2: "))
C=float(input("Valor 3: "))

print("")

if (A>B and C):
    print(f"Maior número: {A}")
    if (B>C):
        print(f"Número do meio: {B}")
        print(f"Menor número: {C}")
    else:
        print(f"Número do meio: {C}")
        print(f"Menor número: {B}")
elif (B>A and C):
    print(f"Maior número: {B}")
    if (A>C):
        print(f"Número do meio: {A}")
        print(f"Menor número: {C}")
    else:
        print(f"Número do meio: {C}")
        print(f"Menor número: {A}")
else:
    print(f"Maior número: {C}")
    if (B>A):
        print(f"Número do meio: {B}")
        print(f"Menor número: {A}")
    else:
        print(f"Número do meio: {A}")
        print(f"Menor número: {B}")

print("")

File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/C/Lista 2 - Sala/5.c
#include <stdio.h>

int main()
{
    float A, B, C;
    
    printf("Ordem crescente\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("Valor 1: "); scanf("%f", &A);
    printf("Valor 2: "); scanf("%f", &B);
    printf("Valor 3: "); scanf("%f", &C);
    
    printf("\n");
    
    if (A>B && A>C) {
        printf("Maior número: %.2f\n", A);
        if (B>C) {
            printf("Número do meio: %.2f\n", B);
            printf("Menor número: %.2f\n", C);
        }
        else {
            printf("Número do meio: %.2f\n", C);
            printf("Menor número: %.2f\n", B);
        }
    }
    else if (B>A && B>C) {
        printf("Maior número: %.2f\n", B);
        if (A>C) {
            printf("Número do meio: %.2f\n", A);
            printf("Menor número: %.2f\n", C);
        }
        else {
            printf("Número do meio: %.2f\n", C);
            printf("Menor número: %.2f\n", A);
        }
    }
    else {
        printf("Maior número: %.2f\n", C);
        if (B>A) {
            printf("Número do meio: %.2f\n", B);
            printf("Menor número: %.2f\n", A);
        }
        else {
            printf("Número do meio: %.2f\n", A);
            printf("Menor número: %.2f\n", B);
        }
    }
}